doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -139,38 +139,92 @@ The powersave level can be specified by using the [option]`radeon_powersave` opt
 For details, see link:http://www.x.org/wiki/RadeonFeature#KMS_Power_Management_Options[www.x.org]. Note that this plug-in is experimental and the option might change in future releases. 
 
 `bootloader`::
-Adds options to the kernel command line. This plug-in supports only the GRUB 2 boot loader.
+Adds options to the kernel command line. This plug-in supports the GRUB 2 boot loader and the Boot Loader Specification (BLS).
 +
 Customized non-standard location of the GRUB 2 configuration file can be specified by the [option]`grub2_cfg_file` option.
 +
 The kernel options are added to the current GRUB configuration and its templates. The system needs to be rebooted for the kernel options to take effect.
 +
-Switching to another profile or manually stopping the `tuned` service removes the additional options. If you shut down or reboot the system, the kernel options persist in the [filename]`grub.cfg` file.
+Switching to another profile or manually stopping the `tuned` service removes the additional options. If you shut down or reboot the system, the kernel options persist in the [filename]`grub.cfg` file and grub environment files.
 +
 The kernel options can be specified by the following syntax:
 +
-[subs=+quotes]
+[subs="+quotes,+macros"]
 ----
-cmdline=[replaceable]_arg1_ [replaceable]_arg2_ ... [replaceable]_argN_
++cmdline+[replaceable]_suffix_=[replaceable]_arg1_ [replaceable]_arg2_ ... [replaceable]_argN_
 ----
 +
+Or with an alternative, but equivalent syntax:
++
+[subs="+quotes,+macros"]
+----
++cmdline+[replaceable]_suffix_=+[replaceable]_arg1_ [replaceable]_arg2_ ... [replaceable]_argN_
+----
++
+Where [replaceable]_suffix_ can be arbitrary (even empty) alphanumeric string which should be unique across all loaded profiles. Good practice is to use the profile name as the [replaceable]_suffix_ (e.g. [option]`cmdline_my_profile`). If there are multiple [option]`cmdline` options with the same suffix (e.g. in the parent profiles), then during the profile load/merge the value which was assigned the last time wins (this is the same as with any other plugin options). Finaly, the kernel command line is constructed by concatenation of the all resulting [option]`cmdline` options.
```

Comment:
Some style changes will be needed here to align the paragraph with our style guide.

Reply:
Unfortunately, I have no idea what to change.